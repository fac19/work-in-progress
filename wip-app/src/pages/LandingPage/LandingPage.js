import React from "react";
import { Link } from "react-router-dom";
import { Button } from "@material-ui/core";
import { LandingUlStyle, LandingLiStyle } from "./LandingPage.style";

const LandingPage = (props) => {
  return (
    <div>
      <h1>Work In Progress</h1>
      <LandingUlStyle>
        <LandingLiStyle>
          <Link to="/log-in">
            <Button variant="contained">Log In</Button>
          </Link>
        </LandingLiStyle>
        <LandingLiStyle>
          <Link to="/sign-up">
            <Button variant="contained">Sign Up</Button>
          </Link>
        </LandingLiStyle>
      </LandingUlStyle>
    </div>
  );
};

export default LandingPage;
