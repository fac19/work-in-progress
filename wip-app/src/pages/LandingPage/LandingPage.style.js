import styled from "styled-components";

const LandingSectionStyle = styled.section`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  text-align: center;
  margin: 2rem;
`;

const LandingUlStyle = styled.ul`
  list-style: none;
  padding: 0;
`;

const LandingLiStyle = styled.li`
  margin: 2rem;
`;

export { LandingUlStyle, LandingLiStyle, LandingSectionStyle };
