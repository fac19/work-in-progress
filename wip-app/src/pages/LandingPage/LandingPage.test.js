import React from "react";
import { createMemoryHistory } from "history";
import TestRenderer from "react-test-renderer";
import LandingPage from "./LandingPage";
import { Router } from "react-router-dom";

describe("Snapshot test", () => {
  it("renders the correct ui", () => {
    const history = createMemoryHistory();
    const component = TestRenderer.create(
      <Router history={history}>
        <LandingPage />
      </Router>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
