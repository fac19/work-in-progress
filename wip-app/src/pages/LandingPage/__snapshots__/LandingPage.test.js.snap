// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test renders the correct ui 1`] = `
<section
  className="sc-AxjAm fpVkNP"
>
  <img
    alt="work in progress logo"
    className="sc-AxhCb kSXmdg"
    src="logo.svg"
  />
  <h1>
    Work In Progress
  </h1>
  <p>
    A social media platform for creatives to receive feedback on their works in progress
  </p>
  <ul
    className="sc-AxirZ cyjTxn"
  >
    <li
      className="sc-AxiKw ddQdlN"
    >
      <a
        className="makeStyles-links-2"
        href="/log-in"
        onClick={[Function]}
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-buttons-1 MuiButton-containedSecondary"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Log In
          </span>
        </button>
      </a>
    </li>
    <li
      className="sc-AxiKw ddQdlN"
    >
      <a
        className="makeStyles-links-2"
        href="/sign-up"
        onClick={[Function]}
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-buttons-1 MuiButton-containedSecondary"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Sign Up
          </span>
        </button>
      </a>
    </li>
  </ul>
</section>
`;
